content/4. Deeltaken/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/Integer/1. Integer & Uint.md

```diff
@@ -0,0 +1,226 @@
+---
+title: 1. Integer
+taxonomie:
+  - ib-19.2.Datatype-Primitief-Integer.DT
+  - ib-19.3.Datatype-Primitief-Integer.DT
+  - ib-19.2.Datatype-Primitief-Uint.DT
+  - ib-19.3.Datatype-Primitief-Uint.DT
+---
+
+> Meer informatie over [[content/2. Ondersteunende-informatie/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/Integer/1. Integer|Integer]]
+
+> Meer informatie over [[2. Uint|Uint]]
+
+## Opdracht 1
+Voeg het initialiseren toe aan het volgende codeblok. Maak gebruik van het integer datatype.
+
+### Specificaties:
+- Maak een variabele `age`.
+- Geef `age` de waarde `23`.
+- Print `age`.
+
+### Verwachte output:
+Als `age = 23, komt in de console:
+```
+De leeftijd is: 23
+```
+
+### Nu jij:
+```csharp runner
+using System;
+
+age = 23;
+
+Console.WriteLine("De leeftijd is: " + age);
+```
+
+> [!info]- Mogelijke uitwerking
+> ``` csharp
+> using System;
+> 
+> int age;
+> 
+> age = 23;
+> 
+> Console.WriteLine("De leeftijd is: " + age);
+> ```
+
+---
+
+## Opdracht 2
+Gebruik bij de volgende casus een integer.
+
+### Specificaties:
+- Maak een integer `saldo` aan.
+- Geef het variabel de waarde `-17`.
+- Print `saldo`.
+
+### Verwachte output:
+Als `saldo` = -17, dan komt in de console:
+```
+Je saldo is: -17
+```
+
+### Nu jij:
+```csharp runner
+using System;
+
+Console.WriteLine("Je saldo is: " + saldo);
+``` 
+
+> [!info]- Mogelijke uitwerking
+> ``` csharp
+> using System;
+>
+> int saldo = -17;
+>
+> Console.WriteLine("Je saldo is: " + saldo);
+> ```
+
+---
+
+## Opdracht 3
+Gebruik bij de volgende casus integer.
+
+### Specificaties
+- Bereken de som van de twee integers.
+- Gebruik hiervoor het `+` operator.
+
+### Verwachte output:
+```
+De som = 6
+```
+
+### Nu jij:
+``` csharp runner
+using System;
+
+int getal1 = 2;
+int getal2 = 4;
+int som;
+
+Console.WriteLine("de som = " + som);
+``` 
+
+> [!info]- Mogelijke uitwerking
+> ``` csharp
+> using System;
+> 
+> int getal1 = 2;
+> int getal2 = 4;
+> int som;
+> 
+> som = getal1 + getal2;
+>
+> Console.WriteLine("de som = " + som);
+> ```
+
+---
+
+## Opdracht 4
+Laat in de volgende casus een integer overflowen.
+
+### Specificaties:
+- Maak een variabele `count`.
+- Vul count met de maximale waarde van een integer (2147483647).
+- Voeg `1` bij count toe.
+- Print count.
+
+### Verwachte output:
+De minimale waarde van een integer:
+```
+-2147483648
```

Comment:
Suggestie: En.... wat als we een int laten 'underflowen' (idk of dat een term is.. bij deze)? De minimale waarde van int pakken, en dan -1 doen?